src/wwwroot/app/services/NoteService.js

```diff
@@ -32,6 +41,19 @@ export default class NoteService {
 
    getNotesByUser() {
       let currentUser = _UserService.getCurrentUser();
-      return _HttpService.get(`${NOTE_URL}/user/${currentUser.id}`);
+      return _HttpService.get(`${NOTE_URL}/user/${currentUser.id}`).then((comments) => {
+         return each(comments, (comment) => {
```

Comment:
it can be:
_HttpService.get(`${NOTE_URL}/user/${currentUser.id}`).then(partialRight(each, this.convertDateFromServerFormat));